packages/@react-spectrum/list/test/ListViewDnd.test.js

```diff
@@ -335,6 +342,196 @@ describe('ListView', function () {
+      });
+
+      it('should allow moving one item into another list', async function () {
+        let {getAllByRole} =  render(<DragBetweenLists />);
+
+        let grid1 = getAllByRole('grid')[0];
+        let grid2 = getAllByRole('grid')[1];
+        let list1rows = within(grid1).getAllByRole('row');
+        let list2rows = within(grid2).getAllByRole('row');
+        let cell = within(list1rows[0]).getByRole('gridcell');
+
+        expect(within(list1rows[0]).getByRole('gridcell')).toHaveTextContent('Item One');
+        expect(within(list1rows[1]).getByRole('gridcell')).toHaveTextContent('Item Two');
+        expect(within(list1rows[2]).getByRole('gridcell')).toHaveTextContent('Item Three');
+
+        expect(within(list2rows[0]).getByRole('gridcell')).toHaveTextContent('Item Seven');
+        expect(within(list2rows[1]).getByRole('gridcell')).toHaveTextContent('Item Eight');
+        expect(within(list2rows[2]).getByRole('gridcell')).toHaveTextContent('Item Nine');
+
+        let dataTransfer = new DataTransfer();
+        fireEvent.pointerDown(cell, {pointerType: 'mouse', button: 0, pointerId: 1, clientX: 0, clientY: 0});
+        fireEvent(cell, new DragEvent('dragstart', {dataTransfer, clientX: 0, clientY: 0}));
+        expect(onDragStart).toHaveBeenCalledTimes(1);
+
+        fireEvent.pointerMove(cell, {pointerType: 'mouse', button: 0, pointerId: 1, clientX: 200, clientY: -1});
+        fireEvent(cell, new DragEvent('drag', {dataTransfer, clientX: 200, clientY: -1}));
+        fireEvent(grid2, new DragEvent('dragover', {dataTransfer, clientX: 200, clientY: -1}));
+        fireEvent.pointerUp(cell, {pointerType: 'mouse', button: 0, pointerId: 200, clientX: 1, clientY: -1});
+
+        fireEvent(cell, new DragEvent('dragend', {dataTransfer, clientX: 200, clientY: -1}));
+        expect(onDragEnd).toHaveBeenCalledTimes(1);
+        
+        fireEvent(grid2, new DragEvent('drop', {dataTransfer, clientX: 200, clientY: -1}));
+        act(() => jest.runAllTimers());
+        await act(async () => Promise.resolve());
+        expect(onDrop).toHaveBeenCalledTimes(1);
+
+        act(() => jest.runAllTimers());
+
+        grid1 = getAllByRole('grid')[0];
+        grid2 = getAllByRole('grid')[1];
+        list1rows = within(grid1).getAllByRole('row');
+        list2rows = within(grid2).getAllByRole('row');
+
+        expect(within(list1rows[0]).getByRole('gridcell')).toHaveTextContent('Item Two');
+        expect(within(list1rows[1]).getByRole('gridcell')).toHaveTextContent('Item Three');
+        expect(within(list1rows[2]).getByRole('gridcell')).toHaveTextContent('Item Four');
+
+        expect(within(list2rows[0]).getByRole('gridcell')).toHaveTextContent('Item One');
+        expect(within(list2rows[1]).getByRole('gridcell')).toHaveTextContent('Item Seven');
+        expect(within(list2rows[2]).getByRole('gridcell')).toHaveTextContent('Item Eight');
+
+        // expect(document.activeElement).toBe(list2rows[0]);
```

Comment:
same as above, the first list is focused here

Reply:
lets put a TODO by it
is this PR supposed to fix this? or what is fixing it?

Reply:
Just added a TODO. The active element is correct in the storybook, but not in the test. Not sure why that would be.

Reply:
I'll take a look

Reply:
The behavior that the test is showing matches the behavior before the change made in this PR. That might be a hint.